networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobSidecarsByRangeMessageHandler.java

```diff
@@ -126,10 +131,11 @@ public void onIncomingMessage(
         peer.getId(),
         message.getCount(),
         startSlot);
+    final UInt64 endSlotFulu = getFuluCappedMaxSlot(endSlot);
```

Comment:
maybe `endSlotBeforeFulu` or other name to reflect that it's not actually Fulu. Or `endSlotAdjusted`